src/Dialogue/Dialogue.tsx

```diff
@@ -5,27 +5,28 @@ const TimesIcon: JSX.Element = <svg name="times" xmlns="http://www.w3.org/2000/s
 
 export interface DialogueProps {
     className?: string;
-    desc?: string;
+    desc?: string | JSX.Element | React.ReactNode;
+    disablePrimaryBtn?: boolean;
```

Comment:
`disablePrimaryBtn` sounds like an action. You should name is `primaryBtnDisabled` instead.

Same for `disableSecondaryBtn`